qpc/insights/tests_insights_upload.py

```diff
@@ -287,11 +303,11 @@ def test_verify_report_success(self):
     def test_verify_report_missing_id(self):
         """Test to verify a QPC report with a missing id is failed."""
         report_json = {
-            'report_type': 'deployments',
+            'report_type': 'insights',
             'report_version': '1.0.0.1b025b8',
             'status': 'completed',
             'report_platform_id': '5f2cc1fd-ec66-4c67-be1b-171a595ce319',
-            'system_fingerprints': [{'key': 'value'}]}
+            'hosts': [{'key': 'value'}]}
```

Comment:
this is kind of a fake test since it won't actually get to the structure of the hosts, but should we make hosts the correct structure (a dictionary)?

Reply:
yep